.travis.yml

```diff
@@ -1,7 +1,11 @@
 sudo: false
 language: node_js
 node_js:
-  - 11
+  - 12
   - 10
-  - 9
   - 8.15.1
+env:
```

Comment:
Please revert the `env` and `before_install` changes, i.e. keep only the Node.js version changes (which could be split BTW).